azure-iot-device/samples/advanced-hub-scenarios/get_storage_sas.py

```diff
@@ -0,0 +1,153 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
>Microsoft [](start = 16, length = 9)

Are you planning to leave this sample here?  It looks like this code is using HTTP directly without going through our client.

Reply:
no this will be removed, but for now I'm going to leave it for testing purposes.

---
In reply to: [352793611](https://github.com/Azure/azure-iot-sdk-python/pull/403#discussion_r352793611) [](ancestors = 352793611)

Reply:
Make sure this is removed

---
In reply to: [353494570](https://github.com/Azure/azure-iot-sdk-python/pull/403#discussion_r353494570) [](ancestors = 353494570,352793611)